### DISCUSSION TITLE
DNS settings

### DISCUSSION THREAD

**Discussion by gofhk** [kaaass/ZerotierFix#88] [Q&A] [answered]
Hi~~~~ 
Zerotierfix is very useful for me.
I wanted to ask can I add another profile for the same Network ID but with different DNS settings ? It is because sometimes I need network DNS sometimes not. At the moment I needed to delete and create profiles to change the DNS settings. Is it possible to make it happen? Thank you very much

**Comment by kaaass** [comment, marked as answer]
Hi, thanks for the suggestion! Is the situation described in #72 similar to yours?

**Reply by gofhk** [reply]
yes, the situation is what I am facing 

Thank you very much

**Comment by kaaass** [comment]
I've just noticed that it might be challenging to have two profiles with the same network ID in the current implementation. Would it be helpful to allow DNS configuration editing after joining the network? This approach would require more clicks but seems much more feasible with our current code base.

The official support for this is likely to be introduced in V2. However, this will be a long-term project since I'm quite busy at the moment 😢.

**Reply by gofhk** [reply]
It is nice to edit the DNS configuration after joining the network. Thank you for your efforts 😊

**Reply by kaaass** [reply]
OK! I will try it~
